app/components/Nav.tsx

```diff
@@ -25,79 +25,200 @@ export default function Widget() {
+                (item) => (
+                  <div
+                    key={item}
+                    className="text-black font-bold text-[1rem] cursor-pointer hover:text-orange-500 transition-colors duration-200 ease-in font-primary"
+                  >
+                    {item}
+                  </div>
+                )
+              )}
+            </div>
+
+            {/* Icons Container */}
+            <div className="hidden md:flex items-center space-x-4 cursor-pointer w-[20%] justify-center">
               <svg
-                width="24px"
-                height="24px"
+                width="22px"
+                height="22px"
                 viewBox="0 0 24 24"
                 fill="none"
                 xmlns="http://www.w3.org/2000/svg"
               >
-                <path
-                  d="M4 7H20M4 12H20M4 17H20"
-                  stroke="#000000"
-                  strokeWidth="2"
-                  strokeLinecap="round"
-                  strokeLinejoin="round"
-                ></path>
+                <g
+                  id="SVGRepo_bgCarrier"
+                  stroke-width="0"
+                ></g>
+                <g
+                  id="SVGRepo_tracerCarrier"
+                  stroke-linecap="round"
+                  stroke-linejoin="round"
+                ></g>
+                <g id="SVGRepo_iconCarrier">
+                  {' '}
+                  <path
+                    d="M16.6725 16.6412L21 21M19 11C19 15.4183 15.4183 19 11 19C6.58172 19 3 15.4183 3 11C3 6.58172 6.58172 3 11 3C15.4183 3 19 6.58172 19 11Z"
+                    stroke="#000"
+                    stroke-width="1.968"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                  ></path>{' '}
+                </g>
               </svg>
-            </button>
-          </div>
-        </div>
-        
-        {/* Mobile Menu */}
-        {isMobileMenuOpen && (
-          <div className="md:hidden flex flex-col items-center bg-white/90 backdrop-blur-md p-4 space-y-2">
-            {['Hero', 'Products', 'Shop', 'Pages', 'Sale', 'Contact'].map((item) => (
-              <div
-                key={item}
-                className="text-black font-bold text-[1rem] cursor-pointer hover:text-orange-500 transition-colors duration-200 ease-in font-primary"
-              >
-                {item}
+              <div className="bg-orange-500 rounded-full p-2 cursor-pointer">
+                <svg
+                  width="22px"
+                  height="22px"
+                  viewBox="0 0 24 24"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <g
+                    id="SVGRepo_bgCarrier"
+                    stroke-width="0"
+                  ></g>
+                  <g
+                    id="SVGRepo_tracerCarrier"
+                    stroke-linecap="round"
```

Comment:
Check how to write stroke-linecap, stroke-linejoin, and stroke-width. Verify the correct way to use these parameters.